git/show.go

```diff
@@ -0,0 +1,115 @@
+package git
+
+import (
+	"fmt"
+	"strings"
+)
+
+type ShowOptions struct {
+	Pretty string
```

Comment:
The man page implies this should have a DiffCommonOptions composed in. I also think "Format" is more descriptive than the Pretty alias.

(Maybe we should have a FormatString type to distinguish it from other types of strings too)